README.md

```diff
@@ -5,7 +5,7 @@ Builds a [SteamOS](http://store.steampowered.com/steamos/) [docker](https://www.
 > ```
 export STEAMUSER_UID=$(id -u ${USER})
 export STEAMUSER_GID=$(id -g ${USER})
-export STEAMUSER_HOME=~${USER}
+export STEAMUSER_HOME=${HOME}
```

Comment:
@rsharo, I think the user of `$USER' also suffers from the same good point you raised about `$HOME'.  What is your thinking here?  I've been going back and forth on the README rewrite, trying to keep things simple, without explicitly encouraging an abbreviated setup that builds a container tightly connected to the local root account.  I think a good intention would be to encourage building a container suitable to be run in the context of the user's normal X session.

Such a trivial doc change has been harder than I expected!  Please let me know your thoughts as I'd like to get this PR over to you.